test/views/AssociateDucrConfirmationViewSpec.scala

```diff
@@ -17,38 +17,55 @@
 package views
 
 import controllers.storage.FlashKeys
-import helpers.views.{AssociateDucrConfirmationMessages, CommonMessages}
+import helpers.views.CommonMessages
 import play.api.mvc.Flash
 import play.twirl.api.Html
-import views.spec.ViewSpec
+import views.html.associate_ducr_confirmation
+import views.spec.{UnitViewSpec, ViewMatchers}
 import views.tags.ViewTest
 
 @ViewTest
-class AssociateDucrConfirmationViewSpec extends ViewSpec with AssociateDucrConfirmationMessages with CommonMessages {
+class AssociateDucrConfirmationViewSpec extends UnitViewSpec with CommonMessages with ViewMatchers {
 
-  private val page = injector.instanceOf[views.html.associate_ducr_confirmation]
+  private val page = new associate_ducr_confirmation(mainTemplate)
+  private val exampleDucr = "5GB123456789000-123ABC456DEFIIIII"
```

Comment:
You can find values for UCRs in `testdata.CommonTestData`

Reply:
Changed in all views